updates/next.json

```diff
@@ -1,7 +1,7 @@
 {
   "stream": "next",
   "metadata": {
-    "last-modified": "2020-04-17T18:59:10Z"
+    "last-modified": "2020-05-07T00:20:07Z"
```

Comment:
tip: I usually just copy/paste the `last-modified` timestamp from the other file (the one generated with `fedora-coreos-stream-generator`)

this is completely optional, though